.babelrc

```diff
@@ -1,6 +1,10 @@
 {
   "presets": [
-    "@babel/preset-env"
+    "@babel/preset-env",
+    "@babel/preset-typescript"
   ],
-  "plugins": ["@babel/plugin-transform-async-to-generator"]
+  "plugins": [
+    "@babel/plugin-transform-async-to-generator",
+    "@babel/plugin-proposal-class-properties"
+  ]
 }
```

Comment:
We can ditch `babel` altogether if using typescript.

Reply:
`yarn start:chrome` threw an error (it said requires plugin-proposal-class-properties) without it for me, so it's there

Reply:
we need to change the webpack configs as well for this to work. replace the `babel-loader` there with `ts-loader`.